### ISSUE TITLE
Not compatible with ruby 2.1.2

### ISSUE THREAD

**Issue by rdunlop** [mailjet/mailjet-gem#43]
Because ruby 2.1.2 doesn't appear to allow `'string': value` syntax, the gem doesn't work.

SyntaxError: .../shared/bundle/ruby/2.1.0/gems/mailjet-1.0.3/lib/mailjet/mailer.rb:47: syntax error, unexpected ':', expecting =>

 'mj-customid': mail['X-MJ-CustomID'] && mai...

Luckily, I was able to upgrade to ruby 2.2.2 to resolve this.

Note: the documentation says that this gem is compatible with 2.x.x.

**Comment by pitch-me-git** [comment]
Hi,

Same here with ruby 2.1.5, but unfortunately our production server won't be upgradable :
Do you have a specific version of the gem for ruby 2.1.5

Thanks!

````
.rvm/gems/ruby-2.1.5/gems/activesupport-4.1.6/lib/active_support/dependencies.rb:247:in `require': /Users/yannkastelnik/.rvm/gems/ruby-2.1.5/gems/mailjet-1.0.3/lib/mailjet/mailer.rb:47: syntax error, unexpected ':', expecting => (SyntaxError)
      'mj-customid': mail['X-MJ-CustomID'] && mai...
````

**Comment by pitch-me-git** [comment]
Update :

Just for info it worked with previous version of the gem (1.0.2)

**Comment by arnaudbreton** [comment]
Hi, 

Thanks for reporting, we're working on a way to fix for previous versions. 
@pitch-me-git: correct, we introduced this variables in 1.0.3. 

Sorry for the inconvenience,
Arnaud.

**Comment by j-mcnally** [comment]
same, ruby 2.1.x doesn't seem to support "key": syntax, would be great to just use => (hashrocket) syntax to be backward compatible, 2.1.x isnt that old.

**Comment by Gormador** [comment]
Fixed in [1.0.4](https://github.com/mailjet/mailjet-gem/releases/tag/1.0.4).

Thank you again, everyone, for reporting the problem and especially @avbrychak for going the extra mile and fixing it.
